@uportal/open-id-connect/src/open-id-connect.js

```diff
@@ -0,0 +1,29 @@
+let bearerToken = null;
+
+export default async function openIdConnec({
+  setTimeout = setTimeout, // allow a custom timeout function to be used
+  userInfoApiUrl = '/uPortal/api/v5-1/userinfo',
+  tokenTimeoutMs = 180000, // 3 minutes
```

Comment:
:arrow_right: In the future I'd like to remove this option and determine timeout based off expiration.

Reply:
That should be reasonably possible the `exp` claim indicates then the token expires.

I imagine there's a potential for issues with clock differences.

Reply:
I was considering adding a configurable clock smear, defaulted to 10ms, to account for differences.